catalog/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path, include
+from catalog.apps import CatalogConfig
+from catalog.views import home, contacts
+
+app_name = CatalogConfig.name
+
+urlpatterns = [
+    path('home/', home, name='home'),
```

Comment:
Домашнюю страницу лучше располагать в корне - чтобы при открытии сайта не приходилось делать лишних переходов
```suggestion
    path('', home, name='home'),
```